spec/features/teachers/account_spec.rb

```diff
@@ -0,0 +1,71 @@
+require "rails_helper"
+
+describe "As a logged-in teacher" do
+  describe "when I visit my account page" do
+    before :each do
+      @teacher = create(:teacher)
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@teacher)
+      visit account_path
+    end
+
+    it "shows me my information" do
+      within ".login-container" do
+        expect(page).to have_content("First Name: #{@teacher.first_name}")
+        expect(page).to have_content("Last Name: #{@teacher.last_name}")
+        expect(page).to have_content("Email: #{@teacher.email}")
+        expect(page).to have_content("Password: ••••••••")
+      end
+    end
+
+    xit "allows me to update my non-password information" do
```

Comment:
This and the below tests all look great.
They will be great for continuing on the `update teacher info` story